infra/bisector.py

```diff
@@ -137,7 +137,8 @@ def _get_dedup_token(output):
 def _check_for_crash(project_name, fuzz_target, testcase_path):
   """Check for crash."""
 
-  def docker_run(args):
+  def docker_run(args, **kwargs):
+    del kwargs
```

Comment:
A local experiment proves that this will not delete variable `architecture` globally, if anyone was worrying about this as I did : )